packages/strapi-types/index.ts

```diff
@@ -18,17 +21,43 @@ type BaseModel = {
   };
 };
 
-type ComponentModel = BaseModel & {};
+export type Model = {
```

Comment:
what model? I am a little bit confused, there are componentModel, baseModel, and model. Maybe componentModel and baseModel is wrong one? or maybe collectionName and uid are present also in all Models -> in that case put it in BaseModel

Reply:
fixed